migrations/1_init/up.sql

```diff
@@ -40,6 +40,7 @@ CREATE TABLE auth.accounts (
     is_anonymous boolean DEFAULT false NOT NULL,
     custom_register_data jsonb,
     otp_secret text,
+    firebase_uid text,
```

Comment:
Why do we have to add `firebase_uid` to the `accounts` table for Firebase, but no other OAuth provider?

Reply:
This is a good point. i will create a separate firebase provider, over the weekend

Reply:
I am not fan with adding columns to the schema that are specific to a provider, is there a way to avoid this?
In any case, if we have to modify the existing schema, please don't change anything in the `1_init` migration, but add a dedicated migration instead. Existing HBP instances will fail to upgrade otherwise.